store/player.ts

```diff
@@ -23,19 +23,26 @@ export const usePlayerStore = defineStore(
       }
     }
 
-    function resetGame(mode: 'tryAgain' | 'exit' = 'exit') {
```

Comment:
Added  the parameter 'resetGame' to get to the route  with the options to quit the game or try again